runtime/src/snapshot_package.rs

```diff
@@ -77,10 +77,17 @@ impl AccountsPackage {
         let snapshot_info = {
             let accounts_db = &bank.rc.accounts.accounts_db;
             let write_version = accounts_db.write_version.load(Ordering::Acquire);
-            // SAFETY: There *must* be an accounts delta hash for this slot.
-            // Since we only snapshot rooted slots, and we know rooted slots must be frozen,
-            // that guarantees this slot will have an accounts delta hash.
-            let accounts_delta_hash = accounts_db.get_accounts_delta_hash(slot).unwrap();
+            let accounts_delta_hash = if bank
+                .feature_set
+                .is_active(&feature_set::remove_accounts_delta_hash::id())
+            {
+                AccountsDeltaHash(Hash::default())
```

Comment:
To confirm, we'll just continue pushing this field (as 0's) to avoid breaking snapshot compat ?